docs/kopia_troubleshooting.md

```diff
@@ -123,4 +114,184 @@ Users may also find the following type of errors in the logs while maintenance i
   * `Error getting a backup store`
   * `BackupStorageLocation is invalid`
 
-     
\ No newline at end of file
+## Kopia Repository Maintenance
+
+* repository maintenance commands:
+```
+kopia maintenance info
+kopia maintenance run 
+kopia maintenance run --full
+```
+
+#### Upstream Documentation
+* Velero - https://velero.io/docs/v1.15/repository-maintenance/
+* Kopia - https://kopia.io/docs/advanced/maintenance/
+
+#### Summary of Kopia Maintenance
+
+* Artifacts in S3 are not deleted by OADP-1.4.x immediately when a backup is deleted.  Due to the safety features of Kopia artifacts will not be deleted until three maintenance cycles has completed which can take up to 72 hours.
+
+#### List of RFE's, bug fix and enhancements
+* https://github.com/vmware-tanzu/velero/issues/8364
+
+#### There are two types of kopia repository maintenance:
+
+* Quick maintenance manages and optimizes indexes and q blobs that store metadata (directory listings, manifests such as snapshots, policies, acls, etc.).
```

Comment:
I would put definition of `q blobs` and `p data blobs` before this paragraph. Below there are many uses of those and it's not very clear what those ares if someone scans just this paragraph.

Reply:
Nah, link works better then repeat for some that will demand more knowledge.